client/components/infinite-list/scroll-helper.js

```diff
@@ -69,8 +68,8 @@ class ScrollHelper {
 		this.forEachInRow(
 			index,
 			function( item ) {
-				const itemKey = this.props.getItemRef( item ),
-					itemBounds = this.boundsForRef( itemKey );
+				const itemKey = this.props.getItemRef( item );
+				const	itemBounds = this.boundsForRef( itemKey );
```

Comment:
Nit: bad whitespace

```suggestion
				const itemBounds = this.boundsForRef( itemKey );
```

Reply:
Huh, I wonder how that got through? Thank you!